node/subsystem-util/src/runtime/mod.rs

```diff
@@ -207,10 +207,10 @@ impl RuntimeInfo {
 	///
 	/// Returns: `None` if not a parachain validator.
 	async fn get_validator_info(&self, session_info: &SessionInfo) -> Result<ValidatorInfo> {
-		if let Some(our_index) = self.get_our_index(&session_info.validators).await {
+		if let Some(our_index) = self.get_our_index(&session_info.validators.to_vec()).await {
```

Comment:
We could update `get_our_index` to work with the new type `Validators` which would also require to implement an iterator interface.